graphql_compiler/compiler/emit_sql.py

```diff
@@ -278,6 +278,7 @@ def _get_xml_path_clause(output_column, predicate_expression):
 )
 
 
+# TODO(bojanserafimov): Rename to FoldSubqueryBuilder and simplify usage and spec.
```

Comment:
@chewselene I'll get to this when you give me the green light. Not excited for a massive merge-conflict :)

Reply:
Good idea! Should be fine to do once #814 is merged